### PR TITLE
Use appassembler for Compute and Dns examples

### PR DESCRIPTION
**Pull request by jean-philippe-martin**
Also, update the corresponding documentation.

### CODE REVIEW

**Comment by jean-philippe-martin** [comment]
CC: @mziccard

File: gcloud-java-examples/src/main/java/com/google/cloud/examples/dns/DnsExample.java [hunk 1]
```diff
@@ -42,21 +42,16 @@
...
- * <p>Steps needed for running the example:
- * <ol>
- * <li>login using gcloud SDK - {@code gcloud auth login}.</li>
- * <li>compile using maven - {@code mvn compile}</li>
- * <li>run using maven - {@code mvn exec:java
- * -Dexec.mainClass="com.google.cloud.examples.dns.DnsExample"
- * -Dexec.args="[<project_id>]
+ * <p>See the README for steps needed for running the example.
+ * <p>The command-line arguments can be:
+ * <PRE>{@code [<project_id>]
```
Comment (jart) [inline gcloud-java-examples/src/main/java/com/google/cloud/examples/dns/DnsExample.java]: Lowercase `<pre>` please. go/htmlstyle

Comment (jean-philippe-martin) [inline gcloud-java-examples/src/main/java/com/google/cloud/examples/dns/DnsExample.java]: done.

**Comment by jart** [comment]
LGTM. Will merge when tests pass.

**Pull request merged**